posthog/batch_exports/models.py

```diff
@@ -147,3 +152,66 @@ def interval_time_delta(self) -> timedelta:
         elif self.interval == "week":
             return timedelta(weeks=1)
         raise ValueError("Invalid interval")
+
+
+class BatchExportLogEntryLevel(str, enum.Enum):
+    DEBUG = "DEBUG"
+    LOG = "LOG"
+    INFO = "INFO"
+    WARNING = "WARNING"
+    ERROR = "ERROR"
```

Comment:
I'm not so sure:
* You can create your own log levels. So other folks may choose to define a custom level that doesn't fit in this enum.
* Javascript doesn't have `WARNING` but `WARN`, so logs generated from there also don't fit the enum.

Reply:
Same as below, I rather not make things generic before we know of other use cases. If this gets duplicated we can abstract it then to avoid the duplication.